packages/rsbuild-plugin-angular/src/lib/plugin/compilation/build-and-analyze.ts

```diff
@@ -31,6 +31,7 @@ import { createFileEmitter } from './file-emitter';
 import { ParallelCompilation } from '@angular/build/src/tools/angular/compilation/parallel-compilation';
 import { normalize } from 'path';
 import { JavaScriptTransformer } from '@angular/build/src/tools/esbuild/javascript-transformer';
+import { NgCompiler } from '@angular/compiler-cli/src/ngtsc/core';
```

Comment:
Let's mark this as a type import as it's only being used for type support
```suggestion
import { type NgCompiler } from '@angular/compiler-cli/src/ngtsc/core';
```

Reply:
Good catch!! 

I can open an issue to setup a lint rule for this!